ac.soton.eventb.statemachines.animation.feature/feature.xml

```diff
@@ -2,7 +2,7 @@
 <feature
       id="ac.soton.eventb.statemachines.animation.feature"
       label="%featureName"
-      version="2.3.1.qualifier"
+      version="2.4.0.qualifier"
       provider-name="%featureVendor"
       plugin="ac.soton.iumlb.branding"
```

Comment:
This branding plug-in is missing. You should create a branding plug-in that use the same iUML-B logo.

Reply:
Branding plugin was not missing - it was a common branding plugin used by all iUML-B features. (see iUML-B utils. git)
However, we decided to adopt separate branding plugins per feature